package.json

```diff
@@ -22,7 +23,11 @@
   "devDependencies": {
     "husky": "^9.0.11",
     "lerna": "^6.6.2",
-    "typescript": "^4.9.3",
-    "pnpm": "^9.12.1"
+    "pnpm": "^9.12.1",
+    "typescript": "^4.9.3"
+  },
+  "volta": {
```

Comment:
Anywhere you see this volta config it can be ignored.  It will only have an effect if you have volta installed and it just does the same thing as nvm (but faster).